CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2021-12-2
+This release is rolling out to all regions - estimated time for completed roll out is 2021-12-2 for public cloud and 2021-12-9 for sovereign clouds.
+
+### Announcement
+
+* Containerd default / Dockershim deprecation on Windows.
+* Konnectivity rollout has been halted for the rest of the year. We will continue the rollout in the new calendar year.
+
+### Release Notes
+
+* Bug Fixes
+  * The DelegateFSGroupToCSIDriver feature flag is only introduced to kubelet in 1.22, the nodes on 1.21 wont be able to start with this feature flag.
+* Component Updates
+  * When cluster has Bring Your Own Key (BYOK) setting, and os disk is ephemeral by default then the fix is to ignore BYOK os disk setting and set os disk as ephemeral. Users could still use BYOK for data disk feature.
+
+## Release 2021-11-25
```

Comment:
I don't think we need to do this. We can probably just roll it all under the 2021-12-2 release

Reply:
Since i had demarcated it will probably leave it like that :) Next time if this scenario comes planning to roll them all into one release.

Reply:
What do you mean? I might just be nitpicking here, but I think it doesn't make sense as we did not have a release on the 25th. Everything is technically rolled into the December 2nd release right? This would technically mean that all the features released on the 25th would be finished rolling out yesterday on public cloud and all sov cloud features would finish on the 9th. We know that is not the case right?